packages/addons/tools/dvb-tools/changelog.txt

```diff
@@ -1,3 +1,7 @@
+107
+- mumudvb: update to 2021-01-06
+- dvblast: update to 2021-01-07
```

Comment:
can you add the change from https://github.com/LibreELEC/LibreELEC.tv/pull/4955

Reply:
Added :-) I didn’t before as there was actually no code change. But better to be complete.